client/src/components/NotFound/NotFound.js

```diff
@@ -1,21 +1,18 @@
 import React from "react";
 import "./NotFound.css";
-import CircularProgress from "@material-ui/core/CircularProgress";
+import Img404 from '../images/404.png';
 
 const NotFound = () => {
-    return (
-        <div class="mainbox">
-            <div class="err">4</div>
-            <CircularProgress className="spinner" />
-            <div class="err2">4</div>
-            <div class="msg">
-                Maybe this page moved? Got deleted? Is hiding out in quarantine?
-                Never existed in the first place?
-                <p>
-                    Let's go <a href="/">home</a> and try from there.
-                </p>
+    return (<div>
+        <h1 className='error_heading1'>
+        Oops &#44; the page you&apos;re looking for doesn&apos;t exist
+      </h1>
+      <img src={Img404} className='error_img' alt='Img404'></img>
+                <h1 className='error_heading2'>
```

Comment:
Please decrease weight of H1 component, it looks weird!

Reply:
Shall i do it to h3?

Reply:
![Screenshot (500)](https://user-images.githubusercontent.com/68965499/116813275-aeaa1100-ab70-11eb-91d5-ce068fd176c0.png)
![Screenshot (501)](https://user-images.githubusercontent.com/68965499/116813276-b10c6b00-ab70-11eb-87c1-9a604a7db61e.png)
@codewithvk is this fine?